### ISSUE TITLE
Question About Offline And Online RL Training

### ISSUE THREAD

**Issue by importpandas** [yuyq18/StepTool#9]
It seems that there is only an offline training code in step_ppo.py & step_ppotraininger.py. The paper mentioned that the step-grained annotated data can be used as offline training or training the reward model.  Are the experimental results in the StepTool paper all from offline methods? Did you implement any online code or results?
